index.html

```diff
@@ -5,6 +5,6 @@
   </head>
   <body>
     <div id="mirador-viewer"></div>
-    <script src="mirador3.min.js"></script>
+    <script src="dist/mirador3.min.js"></script>
```

Comment:
So I think we can revert this change here. The lines here: https://github.com/UCLALibrary/dl-mirador/blob/Restructure_deploy_files/.github/workflows/install-build-deploy.yml#L35-L36 will actually copy the mirador3.js files without the dist directory

Reply:
As an alternative... I could copy the dist into the bucket. Let me push a change to do this